### DISCUSSION TITLE
Will Microsoft.Identity.Web work with ASP . NET web forms with aspx pages?

### DISCUSSION THREAD

**Discussion by sampathcse16** [AzureAD/microsoft-identity-web#1462] [Q&A]
We have an ASP.NET Web Forms application which is targeting 4.7.2.  We are currently using Microsoft.Owin.Security.OpenIdConnect for authenticating users against **Azure AD B2C** . 

Can we use Microsoft.Identity.Web in ASP.NET Webforms application? if you have any example, please share the link with us.

**Comment by jmprieur** [comment]
@sampathcse16 
For the moment, Microsoft.identity.web only provides limited support for OWIN based web apps. See https://github.com/AzureAD/microsoft-identity-web/wiki/asp-net
